app-backend/datamodel/src/main/scala/Scope.scala

```diff
@@ -426,7 +435,12 @@ object Scopes {
   case object StacExportsCRUD
       extends SimpleScope(makeCRUDScopedActions(Domain.StacExports))
 
-  case object TeamsCRUD extends SimpleScope(makeCRUDScopedActions(Domain.Teams))
+  case object TeamsCRUD
+      extends SimpleScope(
+        Set(ScopedAction(Domain.Teams, Action.ListUsers, None)) ++ makeCRUDScopedActions(
```

Comment:
Almost everyone needs to be able to list users on teams that they are on so I think this should be added to basic teams permissions